CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,130 @@
+
+# Contributor Covenant Code of Conduct
```

Comment:
```suggestion
# Contributor Covenant Code of Conduct
```

Reply:
This comes directly from https://github.com/ContributorCovenant/contributor_covenant/blob/release/CODE_OF_CONDUCT.md . I'd prefer to keep the whitespace consistent for future diffs.